src/mcp-bundle/doc/index.rst

```diff
@@ -27,6 +27,23 @@ the ``mcp`` section of your ``config/packages/mcp.yaml`` file.
 To use your application as an MCP server, exposing tools to clients like `Claude Desktop`_, you need to configure in the
 ``client_transports`` section the transports you want to expose to clients. You can use either STDIO or SSE.
 
+**Creating Tools**
+
+Tools are automatically discovered using PHP attributes. Create a class with methods annotated with ``#[McpTool]``::
```

Comment:
```suggestion
Tools are automatically discovered using PHP attributes. Create a class with methods annotated with ``#[AsMcpTool]``::
```
or `#[AsTool]` like in the agent component. Should be a follow-up PR

Reply:
@OskarStark The `#[McpTool]` attribute comes from the official MCP PHP SDK and is defined here:
  https://github.com/modelcontextprotocol/php-sdk/blob/main/src/Capability/Attribute/McpTool.php

  I cannot rename it to `#[AsMcpTool]` in this PR since it's part of the external `mcp/sdk` package. For now, Can I keep `#[McpTool]` in the documentation?